.travis.yml

```diff
@@ -1,37 +1,46 @@
 sudo: required
 dist: trusty
 
+env:
+    global:
+        - ENCRYPTION_LABEL: "79272d66a36b"
+        - COMMIT_AUTHOR_EMAIL: "travis@coala.io"
+
 language: python
 node_js:
-  - "4.1"
+    - "4.1"
 python:
-  - "3.4"
+    - "3.4"
 
 services: docker
 
 cache:
-  pip: true
-  directories:
-    - node_modules
-    - coalahtml/_coalahtml/bower_components
+    pip: true
+    directories:
+        - node_modules
+        - coalahtml/_coalahtml/bower_components
 
 install:
-  - npm install -g bower
-  - npm install
-  - cd coalahtml/_coalahtml
-  - bower install
-  - cd -
-  # colorama needs to be upgraded to fix error when installed colorama
-  # version and required are different.
-  - pip install colorama==0.3.7
-  - pip install -r requirements.txt
-  - pip install -r test-requirements.txt
+    - npm install - g bower
```

Comment:
Hey this is not needed, it just makes some noise on the diff. 
This breaks CI as well due to that space between - and g